### PR TITLE
[Time Conductor] History not working correctly

### LINKED ISSUE #5626: Time Conductor History does not work in real-time mode
#### Summary
I am observing two issues here. The first is the presence of duplicates entries in the time conductor history, which should not appear. The second is that clicking on an historical entry in real-time mode does nothing, the conductor remains in real-time mode with the same offsets.

#### Expected vs Current Behavior
Expected behavior needs clarification with @charlesh88 

#### Steps to Reproduce
1. Load Open MCT
2. In the Time Conductor, switch to real-time mode
3. Click the history button to expand the popup
4. Note that the popup contains duplicate entries
5. Click on history entries
6. Observe that nothing happens in the time conductor

#### Environment
* Open MCT Version: 2.0.7
* Deployment Type: Dev

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [X] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [X] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by jvigliotta**
Closes #5626

### Describe your changes:

Turns out, this was just being caused by a check for fixed vs realtime only happening once on load, moved the computed value to a method (should be safe as this is only updated when the mode changes).

Also, merged in changes from #4414 which has added formatted milliseconds into the duration format, this will prevent the duplicate keys error caused from the menuApi (as names were used for keys).

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: src/plugins/timeConductor/ConductorHistory.vue [hunk 1]
```diff
@@ -66,6 +66,8 @@ export default {
         }
     },
     data() {
+        let isFixedTime = this.openmct.time.clock() === undefined;
```
Comment (ozyx) [inline src/plugins/timeConductor/ConductorHistory.vue:69]: We can remove this declaration and just put it in the return, see below

**Review by ozyx — CHANGES_REQUESTED** [review]
One small change requested, and I found a bug while doing some local testing:

1. Start a fresh instance of Open MCT w/ Local Storage
2. Switch to realtime mode
3. Go to History, you'll see one entry which looks like a Fixed Time history entry erroneously converted to realtime history
4. Click it, see realtime bounds change
5. Switch to Fixed time mode
6. It's 2075, flying cars materialize around you

**Comment by unlikelyzero** [comment]
@jvigliotta we should stub out the expected behavior here.

**Comment by jvigliotta** [comment]
> 

I had written something up, we can compare, sorry was working on fixes!

**Review by ozyx — APPROVED** [review]
LGTM

**Pull request merged**